### DISCUSSION TITLE
Making the multirotor invisible to vehicle cameras

### DISCUSSION THREAD

**Discussion by icisneros** [microsoft/AirSim#3332] [Support Q&A]
In issue #1805 the question was raised as to whether it would be possible to make the drone invisible to the recording cameras.  I'm here to revive that discussion since I face a similar issue. My current simulation setup is a set of six cameras all situated at a point a few inches below the drone.  The idea is to simulate a 360 camera like a GoPro.  Obviously, one of the cameras points upwards to the drone body, and so most of the frame is occluded.  Four of the other cameras that point outward also have some of their frame occluded by the drone body.  The solution proposed by @saihv  in the #1805 thread is technically possible for my situation (would require relocating the six drone cameras equidistant from the center of the drone such that the drone is not in their FOV), but ultimately unsatisfactory because it results in the cameras producing slightly offset images from each other preventing me from seamlessly stitch them together for my intended purpose.

Is it possible to change the ownership of the external camera that follows the drone?  Do any Unreal Engine pros know whether this is a setting in UE, or do I need to modify the FlyingPawn spawning code in Airsim?  I am looking to make the drone invisible to the drone's cameras, but visible to the player (or pilot).  Allowing the external camera to not be owned by the FlyingPawn would allow me to set the "Only Owner See" setting on the drone mesh and solve my problem.

**Comment by ahmed-elsaharti** [comment]
I was going to suggest changing the opacity of the Drone's material to make the drone actually transparent:
![image](https://user-images.githubusercontent.com/49626509/105571454-8ad76380-5d15-11eb-91d8-4bacf37d81ba.png)
then I read that last line where you said you needed the drone to be visible to the player :(
